### DISCUSSION TITLE
How to mount an AWS EFS drive?

### DISCUSSION THREAD

**Discussion by Gregg-Casey** [iterate-ch/cyberduck#15975] [Q&A]
Hello, 
I can't find this information anywhere.  Can I mount an AWS EFS with Mountain Duck?

**Comment by dkocher** [comment]
> Amazon EFS provides … using a traditional file permissions model, file locking, and hierarchical directory structure through the NFS v4 protocol.

We do not currently have any connect option for NFS servers.

**Reply by Gregg-Casey** [reply]
Ok, thanks for the reply @dkocher. 
- I see your "Amazon provides..." note.  Does that imply that there is a work-around?
- Could you add EFS to your plans?
- Recommendation: Add an EFS note to your docs so people know that it is not supported.

**Reply by dkocher** [reply]
I quoted the above from the EFS [documentation](https://aws.amazon.com/efs/features/) to note that the services is implemented as NFS.

**Reply by Gregg-Casey** [reply]
Ok, you mean that AWS implemented it as NFS (not Cyberduck) and so I could mount it without your product as an NFS drive.
I think I saw that people were doing that. Interesting that the EFS also supports SFTP, etc. and there is still no way to use the duck. It would be nice to use your interface to connect.
 "AWS Transfer Family securely scales your recurring business-to-business file transfers to AWS Storage services using SFTP, FTPS, FTP, and AS2 protocols."

Thanks again for your help.

**Reply by dkocher** [reply]
If you put AWS Transfer Family in front of your EFS then you can connect using [Cyberduck](https://docs.aws.amazon.com/transfer/latest/userguide/transfer-file.html#cyberduck) or Mountain Duck.
